apps/innbyggerpanelet/src/app/components/candidateProfile/CandidateIcon.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { ReactElement } from 'react'
+
+export default function CandidateIcon(): ReactElement {
```

Comment:
Why not import icons through the @navikt/ds-icons package, instead of manually? 

https://design.nav.no/designsystem/side/ikoner?icon=People

Reply:
It is the same svg as the navikt ds-icons package, i used the raw svg as then i could controll the size of the icon

Reply:
Will try to do this without using the raw svg. Can see it get quite messy quick if there is a mix of raw svg's and icons that is from the same package